studyhub/base/views.py

```
@@ -72,7 +72,19 @@ def home(request):
 
 def room(request, pk):
     room = Room.objects.get(id=pk)
-    context = {'room': room}
+    room_messages = room.message_set.all().order_by('-created')
+    participants = room.participants.all()
+
+    if request.method == 'POST':
+        message = Message.objects.create(
```

Ruff

Local variable `message` is assigned to but never used

Remove assignment to unused variable `message`

(F841)